[One of the test](https://travis-ci.org/start-jsk/rtmros_hironx/jobs/72435409) is failing. I suspect the error below might be the cause?

```
[ERROR] [WallTime: 1437846364.470115] [19.385000] bad callback: <unbound method TestHiroROSBridgeSend.joint_states_cb>
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/topics.py", line 682, in _invoke_callback
    cb(msg)
TypeError: unbound method joint_states_cb() must be called with TestHiroROSBridgeSend instance as first argument (got JointState instance instead)

[ERROR] [WallTime: 1437846364.470426] [19.385000] bad callback: <unbound method TestHiroROSBridgeSend.joint_states_cb>
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/topics.py", line 682, in _invoke_callback
    cb(msg)
TypeError: unbound method joint_states_cb() must be called with TestHiroROSBridgeSend instance as first argument (got JointState instance instead)

[ERROR] [WallTime: 1437846364.470799] [19.385000] bad callback: <unbound method TestHiroROSBridgeSend.joint_states_cb>
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/topics.py", line 682, in _invoke_callback
    cb(msg)
TypeError: unbound method joint_states_cb() must be called with TestHiroROSBridgeSend instance as first argument (got JointState instance instead)

[ERROR] [WallTime: 1437846364.508696] [19.425000] bad callback: <unbound method TestHiroROSBridgeSend.joint_states_cb>
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rospy/topics.py", line 682, in _invoke_callback
    cb(msg)
TypeError: unbound method joint_states_cb() must 

The log length has exceeded the limit of 4 Megabytes (this usually means that test suite is raising the same exception over and over).

The build has been terminated.
```

--- a/hironx_ros_bridge/test/test_hironx_ros_bridge.py
+++ b/hironx_ros_bridge/test/test_hironx_ros_bridge.py
@@ -45,11 +45,10 @@ class TestHiroROSBridge(unittest.TestCase):
         self.filename_base = tempfile.mkstemp()[1]
         self.filenames = []
 
-        self.robot = hironx.HIRONX()
-        self.robot.init()
-
     @classmethod
     def setUpClass(self):
+        self.robot = hironx.HIRONX()
+        self.robot.init()
 
         self.listener = tf.TransformListener()
 

